wear/src/main/java/io/homeassistant/companion/android/views/ExpandableListHeader.kt

```diff
@@ -40,9 +46,13 @@ fun ExpandableListHeader(
             .clickable { onExpandChanged(!expanded) }
     ) {
         Row {
-            val plusMinus = if (expanded) "-" else "+"
             Text(
-                text = "$string\u2001$plusMinus"
+                text = string
+            )
+            Spacer(modifier = Modifier.width(4.dp))
+            Icon(
```

Comment:
Instead of bundling new XML drawables, you can also use the Material icons that are already bundled with the app in the Android-Iconics library:

```kotlin
Image(
    asset = if (expanded) CommunityMaterial.Icon.cmd_chevron_up else CommunityMaterial.Icon.cmd_chevron_down,
    contentDescription = stringResource(if (expanded) commonR.string.collapse else commonR.string.expand),
    colorFilter = ColorFilter.tint(LocalContentColor.current)
)
```

Reply:
Good suggestion, did not think about it